### ISSUE TITLE
Enhancement request: onPageChanging

### ISSUE THREAD

**Issue by Thunderlemon** [tabalinas/jsgrid#630]
Just a small request to add a hook before the page navigation occurs, a bit like the onItemEditing, onItemInserting work so that you can cancel the event if needed.

**Comment by tabalinas** [comment]
Yes, this would be a good enhancement to add `onPageChanging` callback.
Thank you for the suggestion!
